frontend/src/GoalPage/Goals.jsx

```diff
@@ -2,15 +2,19 @@ import './Goals.css'
 import ProfileTopBar from '../ProfilePage/ProfileTopBar';
 import { useParams } from 'react-router-dom';
 import { useEffect, useState } from 'react';
-import { fetchActiveGoals, fetchAvailableGoals } from '../../HelperFuncs/utils';
+import { fetchActiveGoals, fetchAvailableGoals, fetchTransaction, fetchSimilarUsers, fetchProfile } from '../../HelperFuncs/utils';
```

Comment:
I'll add this an example in my axis doc but to point it out here for now as well `fetchTransaction`, `fetchSimilarUsers` and `fetchProfile` are all three examples of using helpers to avoid duplicate code since these are used in `Profile.jsx` as well